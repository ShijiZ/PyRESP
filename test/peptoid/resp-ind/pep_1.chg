All values are reported in atomic units
%FLAG TITLE
 molecule 1:  peptoid

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
   2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
   3   7.1693530E+00   3.8731120E-01   1.6170360E-01
   4   5.4836410E+00   4.7919210E-01   3.0292610E+00
   5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
   6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
   7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
   8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
   9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
  10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
  11   1.5824870E+00   3.9493260E+00   1.3602580E+00
  12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
  13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
  14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
  15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
  16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
  17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
  18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
  19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
  20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
  21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
  22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          1          0     2.6291139E-02
   2          6          0    -2.4706576E-02
   3          1          0     9.5113652E-02
   4          1          0     7.0046131E-02
   5          6          0     8.5995788E-01
   6          8          0    -1.0267022E+00
   7          7          0    -4.5399725E-01
   8          6          0    -1.8385820E-02
   9          1          0     1.2069734E-01
  10          1          0     8.4429109E-02
  11          1          0     8.4635308E-02
  12          6          0     1.1323568E-02
  13          1          0     1.6985332E-01
  14          1          0     1.3824862E-01
  15          6          0     8.1062506E-01
  16          8          0    -9.4742925E-01
  17          7          0    -8.3005465E-01
  18          1          0     5.7368526E-01
  19          6          0    -1.5944827E-02
  20          1          0     9.1101318E-02
  21          1          0     9.7189580E-02
  22          1          0     8.4023312E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1  -7.0526376E-03  -3.3097893E-02   3.3011031E-04
   2  -1.3745881E-01  -1.4477001E-01  -4.2285628E-02
   3  -1.1164123E-03  -6.9619900E-03  -2.0137988E-03
   4  -8.2912434E-03  -3.2563494E-03   9.0498684E-03
   5  -4.7691313E-01  -6.5057928E-01  -2.2126422E-01
   6  -9.7807518E-02  -4.0814149E-01  -2.5889990E-02
   7  -2.2424421E-02   3.0754766E-02   2.2066618E-02
   8  -9.8155349E-02  -1.8668549E-01   2.3478070E-02
   9  -1.0808471E-02  -1.9343537E-02  -3.7667968E-03
  10  -1.6590961E-03  -1.6002948E-02   3.6420179E-04
  11  -3.5343792E-02  -3.3548633E-02   1.7726452E-02
  12   1.1334748E-01  -6.2057857E-02   2.8812310E-01
  13   1.6665278E-02  -5.6507895E-03   1.9800808E-02
  14   4.2569361E-02  -2.8433362E-02   3.3400486E-02
  15  -1.6186423E-01   1.2502511E-01   7.3331795E-01
  16   9.0519412E-02   9.7780806E-02   3.3570262E-01
  17  -2.0100482E-02   6.3682269E-02   1.9342761E-01
  18   2.0257989E-02   5.7634128E-02   1.7337601E-01
  19   2.3308181E-01   3.0438313E-02  -4.1596428E-02
  20   4.6838443E-02   9.8016183E-03   2.3698753E-03
  21   1.9439332E-02   1.0671351E-03  -2.5396864E-03
  22   1.8871132E-02   1.3678021E-03  -2.8356116E-03

