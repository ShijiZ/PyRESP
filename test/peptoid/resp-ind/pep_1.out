
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
  first stage of RESP. Peptoid for resp charge calculations
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name:  peptoid
 Total charge (ich):  0
 Number of centers: 22
    1    1    0
    2    6    0
    3    1    0
    4    1    0
    5    6    0
    6    8    0
    7    7    0
    8    6    0
    9    1    0
   10    1    0
   11    1    0
   12    6    0
   13    1    0
   14    1    0
   15    6    0
   16    8    0
   17    7    0
   18    1    0
   19    6    0
   20    1    0
   21    1    0
   22    1    0
 ---------------------------------------------------------
 Intra-Molecular Charge Constraint(s) Info for Molecule 1
 ---------------------------------------------------------
    1    1    1    2    1    3    1    4    1    5    1    6
    1   17    1   18    1   19    1   20    1   21    1   22

 -----------------------------
 Atom Ivary Information
 -----------------------------
    1    0
    6    0
    1    0
    1    0
    6    0
    8    0
    7    0
    6    0
    1    0
    1    0
    1    0
    6    0
    1    0
    1    0
    6    0
    8    0
    7    0
    1    0
    6    0
    1    0
    1    0
    1    0

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   22
 Weight factor on initial charge restraints=  0.000500
 There are  3 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    22
 total number of esp points =   983

 center       X               Y               Z
    1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
    2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
    3   7.1693530E+00   3.8731120E-01   1.6170360E-01
    4   5.4836410E+00   4.7919210E-01   3.0292610E+00
    5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
    6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
    7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
    8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
    9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
   10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
   11   1.5824870E+00   3.9493260E+00   1.3602580E+00
   12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
   13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
   14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
   15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
   16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
   17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
   18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
   19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
   20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
   21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
   22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 values will be set to 0

 Number of unique UNfrozen charge centers =    22

 Non-linear optimization requested.
  qchnge =    9.8650341782E-02
  qchnge =    3.1239646967E-03
  qchnge =    1.3230245801E-04
  qchnge =    5.4237968870E-06
  qchnge =    4.1184473577E-07

 Convergence in    4 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1:  peptoid

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         1        0.000000      0.026291      0       0.000000
    2         6        0.000000     -0.024707      0       0.004854
    3         1        0.000000      0.095114      0       0.000000
    4         1        0.000000      0.070046      0       0.000000
    5         6        0.000000      0.859958      0       0.000578
    6         8        0.000000     -1.026702      0       0.000485
    7         7        0.000000     -0.453997      0       0.001076
    8         6        0.000000     -0.018386      0       0.004918
    9         1        0.000000      0.120697      0       0.000000
   10         1        0.000000      0.084429      0       0.000000
   11         1        0.000000      0.084635      0       0.000000
   12         6        0.000000      0.011324      0       0.004968
   13         1        0.000000      0.169853      0       0.000000
   14         1        0.000000      0.138249      0       0.000000
   15         6        0.000000      0.810625      0       0.000612
   16         8        0.000000     -0.947429      0       0.000525
   17         7        0.000000     -0.830055      0       0.000598
   18         1        0.000000      0.573685      0       0.000000
   19         6        0.000000     -0.015945      0       0.004938
   20         1        0.000000      0.091101      0       0.000000
   21         1        0.000000      0.097190      0       0.000000
   22         1        0.000000      0.084023      0       0.000000

 Sum over the calculated charges:      0.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          4.4905295E-01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   2.7799588E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.6816764E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         7.8681096E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        3.82134   -1.10591    2.53965   -2.63244

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   10.15891
        Y    7.63664   -7.81042
        Z    6.12817   -1.60981   -2.34850

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   14.60882
        Y    0.00000   -2.48137
        Z    0.00000    0.00000  -12.12745